src/tests/multihost/alltests/test_misc.py

```diff
@@ -438,16 +438,11 @@ def test_0008_1636002(multihost, backupsssdconf):
         # Try ssh after socket activation is configured
         # Result does not matter we just need to trigger the PAM stack
         with pytest.raises(Exception):
-            check_login_client(multihost, user, 'Secret123')
-
-        # Print pam log for debug purposes
-        multihost.client[0].run_command(
-            'cat /var/log/sssd/sssd_pam.log', raiseonerr=False)
+            check_login_client(multihost, user, 'Secret1234')
```

Comment:
Why password change?

Reply:
I just want to make sure that it fails for sure, as this result does  not matter we just need to trigger the PAM stack

Reply:
ok